cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-logging.adoc

```diff
@@ -0,0 +1,235 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-application-logging"]
+= Tutorial: Logging
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-application-logging
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2024-05-01
+
+There are various methods to view your logs in {product-rosa} (ROSA). Use the following procedures to forward the logs to AWS CloudWatch, view the logs directly through the pod by using `oc logs`, and use LokiStack to search the logs.
+
+[NOTE]
+====
+ROSA is not preconfigured with a logging solution.
+====
+
+The cluster logging components are based on Fluentd and Elasticsearch (ES). Fluentd is deployed to each node in the cluster. It collects application logs and either writes them to ES or forwards them to CloudWatch.
+
+.Prerequisites
+
+* Set up a logging solution before viewing your logs. See the xref:../../observability/logging/cluster-logging-deploying.adoc#cluster-logging-deploying[Installing logging] documentation for more information.
+
+.Additional resources
+* xref:../../observability/logging/cluster-logging.adoc#cluster-logging[Cluster logging]
+
+== Installing the logging add-on service
+
+Install the logging add-on service to forward the logs to AWS CloudWatch.
+
+//These steps were adopted from the Managed OpenShift Black Belt Tutorial "Using the AWS CloudWatch agent to publish metrics to CloudWatch in ROSA".
+
+. Create an `OperatorGroup` object as a YAML file:
++
+.Example `OperatorGroup` object
++
+[source,terminal]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: cluster-logging
+  namespace: openshift-logging 
+spec:
+  targetNamespaces:
+  - openshift-logging
+----
+
+. Run the following script to configure your ROSA cluster to forward logs to CloudWatch:
++
+[NOTE]
+====
+Configuring ROSA to send logs to CloudWatch goes beyond the scope of this tutorial. Integrating with AWS and enabling CloudWatch logging are important aspects of ROSA, so a script has been included to simplify the configuration process. The script will automatically set up AWS CloudWatch. You can examine the script to understand the steps involved.
+====
++
+[source,terminal]
+----
+$ curl https://raw.githubusercontent.com/openshift-cs/rosaworkshop/master/rosa-workshop/ostoy/resources/configure-cloudwatch.sh | bash
+----
++
+.Example output
++
+[source,terminal]
+----
+deployment "cluster-logging-operator" successfully rolled out
+secret/cloudwatch-credentials created
+clusterlogforwarder.logging.openshift.io/instance created
+clusterlogging.logging.openshift.io/instance created
+----
+. After a few minutes, you should begin to see log groups inside of AWS CloudWatch. Run the following command to see the log groups:
++
+[source,terminal]
+----
+$ aws logs describe-log-groups --log-group-name-prefix rosa-${GUID}
```

Comment:
GUID is no longer being used.  This should now be "rosa-<cluster_name>".

So in full:
`aws logs describe-log-groups --log-group-name-prefix rosa-mycluster`